inc/namespace.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Altis\Chassis;
```

Comment:
This should match the load.php's namespace:

```suggestion
namespace Altis\Local_Chassis;
```

Reply:
This was done to match the other files in this module, I'll switch the other files over as well